tests/test_browser.py

```diff
@@ -4311,7 +4311,7 @@ def test_small_js_flags(self):
     size = os.path.getsize('test.js')
     print('size:', size)
     # Note that this size includes test harness additions (for reporting the result, etc.).
-    self.assertLess(abs(size - 5368), 100)
+    self.assertLess(abs(size - 5477), 100)
```

Comment:
why is this still needed?

Reply:
This is purely due to the increase in the code size of the JS error reporting code used by the tests.   Sadly I can't see how to avoid including that in the measured code size, even though we don't care about it.